packages/chatgpt/src/Adaptor.js

```diff
@@ -0,0 +1,61 @@
+import {
+  execute as commonExecute,
+  composeNextState,
+} from '@openfn/language-common';
+import { expandReferences } from '@openfn/language-common/util';
+import { request } from './Utils';
+import 'dotenv/config';
+
+import OpenAI from 'openai';
+
+/**
+ * Prompt OpenAI model.
+ * @public
+ * @example
+ * promptChatGpt("prompt")(state)
+ * @function
+ * @param {string} prompt - User textual prompt
+ * @returns {Operation}
+ */
+export function promptChatGpt(prompt) {
```

Comment:
A slight suggestion on the function signature that will give users more flexibility 
```
@param {string} prompt - User textual prompt
@param {object} options - Options for textual prompt eg model, role
promptChatGpt(prompt, options)
```